### PR TITLE
Added category cards to design system

### PR DESCRIPTION
**Pull request by mattsskea**

### CODE REVIEW

File: components/container-cards/category-card.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  &:hover {
+    .msds-container-card-category__header {
+      color: $yellow;
+    }
+  }
+
+  &:active {
+    .msds-container-card-category__header {
+      box-shadow: 0 0 0 4px rgba(255, 255, 255, 0.5);
```
Comment (ms-Danny) [inline components/container-cards/category-card.scss]: Should this shadow come from a variable?

Comment (mattsskea) [reply components/container-cards/category-card.scss]: We dont have a variable setup for rgba(255, 255, 255, 0.5)

File: components/container-cards/category-card-dark.config.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+collated: true,
+status: 'prototype'
```
Comment (mmussmann) [inline components/container-cards/category-card-dark.config.yml]: I think you should flag it as "wip", a prototype should not be implemented anywhere until it has moved on to be either wip or ready.

File: docs/05-icons.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 title: Icons
 ---
 ### Adding a new icon
+- Use SVGOMG to optimize the SVG https://jakearchibald.github.io/svgomg/
```
Comment (mmussmann) [inline docs/05-icons.md]: It will also be done automatically by the build step so not critical. (But it doens't hurt :))

**Pull request merged**